### PR TITLE
Adds a nifty new fossil type

### PR DESCRIPTION
**Pull request by MariahShiri**
Adds a new type of fossil!!

### CODE REVIEW

**Comment by MariahShiri** [comment]
!review

File: code/modules/research/xenoarchaeology/finds/finds_fossils.dm [hunk 1]
```diff
@@ -46,16 +46,19 @@
...
+	var/list/skeleton_types = list("biped", "beast")
 	var/breq
 	var/bstate = 0
 	var/plaque_contents = "Unnamed alien creature"
 
 /obj/skeleton/New()
 	src.breq = rand(6)+3
 	src.desc = "An incomplete skeleton, looks like it could use [src.breq-src.bnum] more bones."
+	skeleton_type = pick(skeleton_types)
+	icon_state = "uskel_[skeleton_type]"
```
Comment (alsoandanswer) [inline code/modules/research/xenoarchaeology/finds/finds_fossils.dm:61]: I'm pretty you've missed porting some sprites here.

Comment (MariahShiri) [reply code/modules/research/xenoarchaeology/finds/finds_fossils.dm:61]: wym

**Comment by NonQueueingMatt** [comment]
!review

**Review by JohnWildkins — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
Needs another commit to kickstart the unit tests.

**Pull request closed without merging**